archstrike-testing/bluez-utils-compat/PKGBUILD

```diff
@@ -0,0 +1,65 @@
+# Maintainer: David Thurstenson <thurstylark@gmail.com>
+# Contributor: Tom Gundersen <teg@jklm.no>
+# Contributor: Andrea Scarpino <andrea@archlinux.org>
+# Contributor: Geoffroy Carrier <geoffroy@archlinux.org>
+
+buildarch=220
+
+_pkgname='bluez'
+pkgname='bluez-utils-compat'
+pkgver=5.47
+pkgrel=1
+url="http://www.bluez.org/"
+arch=('i686' 'x86_64' 'armv5h' 'armv6h' 'armv7h' 'aarch64')
+license=('GPL2')
+pkgdesc="Development and debugging utilities for the bluetooth protocol stack. Includes deprecated tools."
+depends=('dbus' 'systemd' 'glib2')
+conflicts=('bluez-hcidump' 'bluez-utils')
+provides=('bluez-hcidump' 'bluez-utils')
+replaces=('bluez-hcidump' 'bluez<=4.101')
```

Comment:
not sure if we want to do this

Reply:
the conflicts w/ bluez-utils makes sense because it's in extras. provides makes sense. the replaces I'm indifferent to.